apps/condo/domains/ticket/components/TicketId/TicketResidentFeatures.tsx

```diff
@@ -50,37 +51,52 @@ const getIsPaymentsInMobileAppAvailable = (receiptsInCurrentPeriod, receiptsInPr
     return false
 }
 
-const PaymentsAvailableIndicator: React.FC<PaymentsAvailableIndicatorProps> = ({ ticketOrganizationId }) => {
+const TWO_MONTHS_AGO = dayjs().startOf('day').subtract(60, 'days').toISOString()
+
+const PaymentsAvailableIndicator: React.FC<PaymentsAvailableIndicatorProps> = ({ ticketOrganizationId, propertyAddress }) => {
     const intl = useIntl()
     const PaymentsAvailableMessage = intl.formatMessage({ id: 'pages.condo.ticket.PaymentsAvailable' })
     const PaymentsNotAvailableMessage = intl.formatMessage({ id: 'pages.condo.ticket.PaymentsNotAvailable' })
 
     const currentPeriod = dayjs().startOf('month').subtract(1, 'month').format('YYYY-MM-DD')
     const previousPeriod = dayjs(currentPeriod).subtract(1, 'month').format('YYYY-MM-DD')
 
-    const receiptsSearchQuery = { context: { organization: { id: ticketOrganizationId, deletedAt: null }, deletedAt: null } }
+    const receiptsSearchQuery = { context: { organization: { id: ticketOrganizationId } } }
 
     const { count: receiptsInCurrentPeriod, loading: currentPeriodLoading } = BillingReceipt.useCount({
         where: {
             ...receiptsSearchQuery,
             period: currentPeriod,
-            deletedAt: null,
         },
     })
 
     const { count: receiptsInPreviousPeriod, loading: previousPeriodLoading } = BillingReceipt.useCount({
         where: {
             ...receiptsSearchQuery,
             period: previousPeriod,
-            deletedAt: null,
         },
     })
 
+    const { count: receiptsByProperty, loading: receiptsByPropertyLoading } = BillingReceipt.useCount({
+        where: {
+            property: {
+                OR: [
+                    {
+                        address: propertyAddress,
+                        normalizedAddress: propertyAddress,
```

Comment:
Why there are two same values used for search by different values? Can it be done only by one of provided fields?

Reply:
As long as there is no address service, the normalizedAddress field is empty.
After the address normalization logic appears, it will be possible to remove the filter by the address field and fallback to the old logic

Reply:
But is that work correct? Maybe it should be done like this:
```js
...
OR: [
    { address: propertyAddress, },
    { normalizedAddress: propertyAddress, },
]

```

Reply:
done